package.json

```diff
@@ -19,6 +19,7 @@
     "lint": "eslint {src,tests}/**/*.{ts,tsx}",
     "lint:fix": "yarn lint --fix",
     "lint:types": "tsc --noEmit",
+    "lint:prettier":"yarn prettier --write src/**/**/*.{ts,tsx}",
```

Comment:
Not sure that `yarn` call is needed here. Since scripts are called anyway from the PM environment there is no need to hard bind the manager in most cases.

Reply:
```suggestion
    "lint:prettier":"prettier --write src/**/**/*.{ts,tsx}",
```